owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/model/RangeModel.java

```diff
@@ -0,0 +1,65 @@
+package fr.openwide.core.wicket.more.model;
+
+import org.apache.wicket.model.AbstractReadOnlyModel;
+import org.apache.wicket.model.IModel;
+
+import com.google.common.collect.Range;
+
+import fr.openwide.core.wicket.more.util.model.Detachables;
+
+/**
+ * Creates a range model representing an interval between two values.
+ */
+public class RangeModel<C extends Comparable<?>> extends AbstractReadOnlyModel<Range<C>> {
+
+	private static final long serialVersionUID = 2798667461907515108L;
+	
+	private final IModel<C> lowerBoundModel;
+	private final IModel<C> upperBoundModel;
+	
+	/**
+	 * Constructs a closed range model from a lower and an upper bound.
+	 * 
+	 * @param lowerBoundModel
+	 *         the lower bound of this range, or null for negative infinity
+	 * @param upperBoundModel
+	 *         the upper bound of this range, or null for positive infinity
+	 */
+	public RangeModel(IModel<C> lowerBoundModel, IModel<C> upperBoundModel) {
+		super();
+		// NOTE Penser à surcharger avec des boolean si besoin de manipuler des intervalles ouverts / semi-ouverts.
+		this.lowerBoundModel = lowerBoundModel;
+		this.upperBoundModel = upperBoundModel;
+	}
+	
+	/**
+	 * Constructs a closed range model representing a point where the lower bound and the upper bound are equal.
+	 * 
+	 * @param pointModel
+	 *         the point representing both the lower and the upper bounds
+	 */
+	public RangeModel(IModel<C> pointModel) {
```

Comment:
A nice enhancement would be to make the constructors private and instead provide static methods for creating range models. You could use the same method names as in `Range`: 

 * `closed(IModel<C>, IModel<C>)`
 * `atMost(IModel<C>)`
 * `atLeast(IModel<C>)`
 * `singleton(IModel<C>)`

The other types of ranges (open/half-open) are not needed yet, so let's just leave them alone for now.